src/main/asciidoc/index_cn.adoc

```diff
@@ -142,154 +141,176 @@ image::ApacheIncubator.png[]
 * Mentor for IoTDB, Dubbo, RocketMQ and Apex
 
-== The Apache Incubator
+== Apache孵化器
 [.twocolumns]
 --
-* Learn The Apache Way
-* Build the Community
-* Establish Licensing
-* Integrate Infrastructure
-* Release
+* 理解Apache之道
+* 发展社区
+* 建立协议
+* 发布
 
 image::bridge.jpg[float=right]
 --
 
-== The Apache Incubator
+== Apache孵化器
 [.twocolumns]
 --
-* Takes 1-2 years sometimes longer
-* Graduate to a Top Level Project
-* Not all projects graduate
+* 需要1-2年或更久的时间
+* 逐渐成为顶级项目
+* 并非每一个项目都有这样的机会
 
 image::bike.jpg[float=right]
 --
 
-== Incubation: Build the Community
-* **Users** — Users are the reason a project exists. No users ⇒ no project
-* **Contributors** — Our projects depend on contributions from the community -- bug reports, email discussions, bug fixes, documentation
-* **Committers** — Frequent contributors become committers after demonstrating merit
-* **Project Management Committee Members** — The PMC is responsible for setting the direction of the project, authorizing releases, and voting new committers and PMC members
-
-== Community
-* **Politeness** — Be nice
-* **Respect** — Everyone’s point of view has value
-* **Trust** — Assume the best intentions
-* **Humility** — Others might have better ideas
-
-Code of conduct: https://www.apache.org/foundation/policies/conduct.html
-
-== Incubation: Licensing is a key aspect
-* **Provenance** — Establish where every line of code originated
-* **Trademarks** — Must not conflict with existing names
-* **Software Grants** — The owner of the intellectual property of existing code/test/documentation must grant rights to Apache
-* **Contributor Licenses** — Contributions while at Apache are governed by contributor licenses:
-** Copyright
-** Patent
-** Distribution/Modification
-
-
-== Infrastructure: Keeping the wheels on
-* **Mail lists** — “If it isn't on an archived list, it didn't happen”. Communication and review are essential to project governance.
-* **Source repository** — Viewing all parts of the system is open to the public. Changes are made only by committers via procedures established by the PMC.
-* **Build/Test**  — Most projects now incorporate continuous integration build and test processes.
-* **Establish a Web Presence** — Each project has its own unique web page https://project.apache.org
-
-== Release distributions
-* Legally an act of the Foundation
-** “Does NOT need to work”
-** “Does need proper licensing”
-* Authorized by the Project Management Committee
-** Vote required
-** Digital Signature(s) required
-* Distributed via extensive mirror system
-
-== Incubation: Graduation
+== 社区
+* **友善** — 友善待人
+* **尊重** — 每个人的意见都有价值
+* **信任** — 假设每个人都抱有最大的善意
+* **谦虚** — 其他人可能会有更好的想法
+
+
+[.small]#Code of conduct: https://www.apache.org/foundation/policies/conduct.html#
+
+== 构建协议是关键
+* **起源** — 当代码被创造出来时，需要建立协议
+* **商标** — 不可以与已经存在的名称有冲突
+* **软件许可** — 现有代码，测试，文档需要知识产权的拥有者需要将软件许可赋予Apache
+* **贡献者协议** — 在Apache的贡献需要遵循贡献者协议：
+** 版权， 专利， 分发/修改
+
+== Apache协议
+* ？Universal donor 无门槛
```

Comment:
这块的意思我也有点不清楚。
@justinmclean  Could you explain the meaning of "Universal donor"? 
We want to find a right way to translate it.

Reply:
OK, I just did a quick search about it on google.
"For emergency transfusions, blood group type O negative blood is the variety of blood that has the lowest risk of causing serious reactions for most people who receive it. Because of this, it's sometimes called the universal blood donor type."